docs/_posts/examples/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "MapboxGeocoder": true,
     "MapboxDirections": true,
     "turf": true,
-    "d3": true
+    "d3": true,
+    "Uint8Array": true
```

Comment:
@jfirebaugh as far as I remember, the `env` switches are only for predefined globals like `Uint8Array` — they're not used for catching keywords like `let`.